Formula/httpd.rb

```diff
@@ -18,14 +18,188 @@ class Httpd < Formula
   depends_on "pcre"
 
   def install
-    # use Slackware-FHS layout as it's closest to what we want.
-    # these values cannot be passed directly to configure, unfortunately.
-    inreplace "config.layout" do |s|
-      s.gsub! "${datadir}/cgi-bin", "#{pkgshare}/cgi-bin"
-      s.gsub! "${datadir}/error",   "#{pkgshare}/error"
-      s.gsub! "${datadir}/htdocs",  "#{pkgshare}/htdocs"
-      s.gsub! "${datadir}/icons",   "#{pkgshare}/icons"
-    end
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+
+    # fix default user/group when running as root
+    inreplace "docs/conf/httpd.conf.in", /(User|Group) daemon/, "\\1 _www"
+
+    Pathname.new("docs/conf/httpd.conf.in").append_lines <<-EOS.undent
```

Comment:
Is `Pathname.new` actually required here?

Reply:
rhetorical question, much?

Reply:
You know my tone too well 🙈.

Reply:
@kabel `(buildpath/"docs/conf/httpd.conf.in").append_lines` (and similar format elsewhere)

Reply:
Ah, I knew there had to be a variable I was missing.

Reply:
@MikeMcQuaid My bad, sorry, should've included that information with my comment.

Reply:
Let's pass on this for now.